hash/hash_blake2b.c

```diff
@@ -0,0 +1,343 @@
+/**
+ * @addtogroup hash Hash algorithms
+ * @{
+ * @file hash_blake2b.c
```

Comment:
No need to add the filename. It's best to let Doxygen automatically detect it, otherwise, you'd have to specify the whole directory as well.
```suggestion
 * @file
```